.changeset/ready-windows-switch.md

```diff
@@ -0,0 +1,5 @@
+---
+"@fluidframework/container-runtime": minor
+---
+
```

Comment:
```suggestion
---
"@fluidframework/container-runtime": minor
---
---
section: deprecation
---
```

Upcoming build-tools changes will collect this metadata automatically.

Reply:
Changed it